### PR TITLE
Milestone: contact form for my portfolio web page mobile version

### PR DESCRIPTION
**Pull request by Kashif-Saleem-Ghuman**
**In this milestone:**

- I have added a contact me section and footer section providing links to my social media accounts
- I have updated the style.css sheet for styling the contact and footer section
- I have also made the page responsive by making changes to the style.css file

### CODE REVIEW

**Review by emmiiorji — COMMENTED** [review]
Hi @Kashif-Saleem-Ghuman, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a a new pull request for a re-review. Kindly note that it is required to not create a new pull request for a second review. Please, make the changes required by the previous reviewer and submit the same pull request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Kashif-Saleem-Ghuman** [comment]
@emmiiorji I have submitted this the **FIRST** time for the **FIRST** Review

**Comment by emmiiorji** [comment]
Hi @Kashif-Saleem-Ghuman,

Sorry about that. Kindly resubmit and the next reviewer will do justice.

**Comment by emmiiorji** [comment]
Hi @Kashif-Saleem-Ghuman,

Did you mean you had earlier submitted this pull request for a first review? However, I can't seem to find the first reviewer's comments.

**Comment by Kashif-Saleem-Ghuman** [comment]
> Hi @Kashif-Saleem-Ghuman,
> 
> Sorry about that. Kindly resubmit and the next reviewer will do justice.

@emmiiorji That's because this is the first time I am submitting **THIS** project for a review. 
However yesterday I submitted day one's exercise in **this** submission by mistake, and I informed the support team and they manually undid that submission from here.
 But I believe the database is still considering that one submitted since it is showing that I have 2 reviews out of 3 left. So it is like counting that mistaken submission as well. I have already emailed the support team about this issue.

File: index.html [hunk 1]
```diff
@@ -18,174 +18,211 @@
...
+      <div class="corner__geometry">
+         <img src="./images/corner_geometry.svg">
+      </div>
+
+      <h2 class="h2">Get Started</h2>
+      <p class="contact__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+    <form action="https://formspree.io/f/xeqdpkpy" method="post" class="contact__form"> <!-- change made -->
+      <input type="text" required maxlength="30" placeholder="your name" name="name" class="name">
+      <input type="email" required pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" placeholder="your email" name="email" class="email">
+      <textarea type="text" placeholder="Enter your message here..." name="message" id="" class="message"  required> </textarea>
```
Comment (1GORDON) [inline index.html]: Great job @Kashif-Saleem-Ghuman 🤝

- [ ] As stated in the project requirements, kindly add the `maxlength="500"` attribute to the Textarea field.
- [ ] Remember to remove this `maxlength="500"` attribute from the submit button when you seem to have mistakenly implemented it.

File: index.html [hunk 2]
```diff
@@ -18,174 +18,211 @@
...
+  
+
+  
+    <section class="contact">
+      <div class="corner__geometry">
+         <img src="./images/corner_geometry.svg">
+      </div>
+
+      <h2 class="h2">Get Started</h2>
+      <p class="contact__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
```
Comment (1GORDON) [inline index.html:200]: Great job @Kashif-Saleem-Ghuman 🤝

- [ ] As stated in the project requirements, kindly align the elements of the contact form with `display: flex`.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Kashif-Saleem-Ghuman ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Linters are passing ✔️
- Descriptive Pull request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] Kindly add a correct Readme file and customize it accordingly. You may use this [readme template](https://github.com/microverseinc/readme-template) to get started.

_Check the comments under the review._

## Optional suggestions

- [ ] You may maintain your commit messages in an imperative tense, something like `Add Image` instead of `Added Image`.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -18,174 +18,211 @@
...
+    <form action="https://formspree.io/f/xeqdpkpy" method="post" class="contact__form"> <!-- change made -->
+      <input type="text" required maxlength="30" placeholder="your name" name="name" class="name">
+      <input type="email" required pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" placeholder="your email" name="email" class="email">
+      <textarea type="text" maxlength="500" placeholder="Enter your message here..." name="message" id="" class="message"  required> </textarea>
+      <input type="submit" value="Start Collaboration" class="buttons contact__button">
+     </form>
```
Comment (Bondok6) [inline index.html:206]: - [OPTIONAL] I noticed that the font-size of the submit button is too large, so I suggest decreasing the font-size of the button to match your Figma design 💯

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @Kashif-Saleem-Ghuman 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Great job implementing the changes provided by the previous reviewers ✅
- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- No Linter errors ✅


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**